testing/test_deploy_app.py

```diff
@@ -179,6 +217,105 @@ def get_google_open_id_connect_token(service_account_credentials):
     request, OAUTH_TOKEN_URI, body)
   return token_response['id_token']
 
+def delete_gcloud_resource(args, keyword, filter='', dlt_params=[]):
+  get_cmd = 'gcloud compute %s list --project=%s --format="value(name)"' % (keyword, args.project)
```

Comment:
Add a TODO to replace this with use of the client libraries rather than shelling out to gcloud.